src/constants/errors.ts

```diff
@@ -0,0 +1,23 @@
+export const ERRORS = {
```

Comment:
Decided to start extracting potential error messages into its own file as in some cases we use the same messaging